atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -118,6 +123,33 @@ public void create(byte[] tableMetadata) {
                 "INSERT INTO " + config.metadataTable().getQualifiedName() + " (table_name, table_size) VALUES (?, ?)");
     }
 
+    private void maybeAlterTableToHaveOverflow() {
+        try {
+            String shortTableName = oracleTableNameGetter.getInternalShortTableName(conns, tableRef);
+
+            if (config.alterTablesOrMetadataToMatch().contains(tableRef)
+                    && overflowTableHasMigrated()
+                    && overflowTableExists()
+                    && !overflowColumnExists(shortTableName)) {
+                alterTableToHaveOverflowColumn(shortTableName);
+            }
+        } catch (TableMappingNotFoundException e) {
+            log.warn("Table mapping expected but not found for table.", UnsafeArg.of("tableRef", tableRef), e);
```

Comment:
This is kind of ugly but caught in other integration tests where `createTable` is tested when having mapping configured vs not configured. It may be worth (outside of this PR maybe?)  to throw on this particular condition (i.e., table is created, we expect a mapping, but one is not found).

Reply:
Just to clarify, the bad case here is: We have table mapping enabled, we create a table, but there's no mapping for that table? Yeah I love those cases.
I think it's safe to just throw (use Throwables.rewrapAndThrowUncheckedException or similar), since you're already in a bad state but the admin is actively trying to remediate, so this will flag that the remediation did not happen successfully and why. 
Also, you use LoggingArgs for table ref, you should be able to use it here too.

Reply:
Separately, we can just scope the try catch to just the bit that can actually throw the table mapping not found exception, instead of everything.

Reply:
`Just to clarify, the bad case here is: We have table mapping enabled, we create a table, but there's no mapping for that table? Yeah I love those cases.`
Correct! 

`the admin is actively trying to remediate, so this will flag that the remediation did not happen successfully and why.` I agree with this, but we're potentially not performing remediation (we perform the mapping _before_ we even check to see if we need to alter these tables). We, of course, can move the code around to better accommodate this, but this will just break elsewhere in the code.